### PR TITLE
PlottingGL: PCP fix

### PR DESCRIPTION
**Pull request by martinfalk**
* fixed a crash caused by refactoring in 547e4a6

### CODE REVIEW

**Review by petersteneteg — APPROVED** [review]

File: modules/plottinggl/src/processors/parallelcoordinates/parallelcoordinates.cpp [hunk 1]
```diff
@@ -342,6 +342,7 @@ void ParallelCoordinates::createOrUpdateProperties() {
     if (!data->getNumberOfRows()) return;
     if (!data->getNumberOfColumns()) return;
 
+    updating_ = true;
```
Comment (r-englund) [inline modules/plottinggl/src/processors/parallelcoordinates/parallelcoordinates.cpp:345]: use util::KeepTrueWhileInScope  (see my same fix in https://github.com/inviwo/inviwo/commit/160dcf73ea35d0aa847cc814bc3f3ef3cdffc69f)

**Review by r-englund — CHANGES_REQUESTED** [review]
> fixed a crash caused by refactoring in 547e4a6

There was an issue with this before that commit, but it weren't crashing

File: modules/plottinggl/src/processors/parallelcoordinates/parallelcoordinates.cpp (hunk 1 above)
Comment (martinfalk) [reply modules/plottinggl/src/processors/parallelcoordinates/parallelcoordinates.cpp:345]: there is some code at the end of the function which requires the bool to be reverted to false again. When using KeepTrueWhileInScope one has to do that anyway or introduce another `{ }` block just for the raii thingy.

**Comment by martinfalk** [comment]
It was for me in case the input dataframe was modified.

**Review by r-englund — APPROVED** [review]

File: modules/plottinggl/src/processors/parallelcoordinates/parallelcoordinates.cpp (hunk 1 above)
Comment (r-englund) [reply modules/plottinggl/src/processors/parallelcoordinates/parallelcoordinates.cpp:345]: I see

**Pull request merged**